sdk/keyvault/keyvault-admin/src/accessControlClient.ts

```diff
@@ -89,7 +89,7 @@ export class KeyVaultAccessControlClient {
     this.client.pipeline.addPolicy(
       bearerTokenAuthenticationPolicy({
         credential,
-        scopes: authenticationScopes,
+        scopes: [],
```

Comment:
This change sounds familiar, @timovv was it part of one of your PRs?

Reply:
@deyaaeldeen Yes, @timovv was addressing it in a different PR in case this one took more time to get merged. But we find the issue with the auth scopes 2 weeks ago while working on fixing the tests for this PR.